assets/optional/multus/06-daemonset.yaml

```diff
@@ -0,0 +1,114 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: multus
+  namespace: openshift-multus
+  annotations:
+    kubernetes.io/description: |
+      This daemonset launches the Multus networking component on each node.
+spec:
+  selector:
+    matchLabels:
+      app: multus
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        app: multus
+        component: network
+        type: infra
+        openshift.io/component: network
+        kubernetes.io/os: "linux"
+    spec:
+      serviceAccountName: multus
+      hostNetwork: true
+      nodeSelector:
+        kubernetes.io/os: linux
+      priorityClassName: "system-node-critical"
+      tolerations:
+      - operator: Exists
+      initContainers:
+      - name: install-container-network-plugins
+        image: container-networking-plugins
+        imagePullPolicy: IfNotPresent
+        command: [ "/bin/bash", "-ec", "--" ]
+        args:
+          - >
+            /entrypoint/cnibincopy.sh;
+        volumeMounts:
+        - name: cnibin
+          mountPath: /host/opt/cni/bin
+        - mountPath: /entrypoint
+          name: cni-binary-copy
+        env:
+        - name: DEFAULT_SOURCE_DIRECTORY
+          # TODO: Change when microshift-specific multus image is ready
+          value: "/usr/src/plugins/rhel9/bin/"
+        - name: BINARIES_TO_COPY
+          value: "{bridge,ipvlan,macvlan,static,dhcp,host-local}"
+      containers:
+      - name: kube-multus
+        image: multus-cni
+        imagePullPolicy: IfNotPresent
+        command: [ "/bin/bash", "-ec", "--" ]
+        args:
+          - >
+            /entrypoint/cnibincopy.sh;
+            exec /usr/src/multus-cni/bin/thin_entrypoint
+            --multus-conf-file=auto
+            --multus-autoconfig-dir=/host/etc/cni/net.d
+            --readiness-indicator-file=/etc/cni/net.d/10-ovn-kubernetes.conf
```

Comment:
`--readiness-indicator-file=/etc/cni/net.d/10-ovn-kubernetes.conf` expects that ovn-k config will be created under '/etc/cni/net.d' of multus pod. Is that actually created by ovn-k?

Reply:
Is it multus pod or multus cni (/opt/cni/bin/multus)? But yeah, we have that file on the host ([ref](https://github.com/openshift/microshift/blob/main/assets/components/ovn/single-node/master/daemonset.yaml#L376))